docs/usage/configuration-options.md

```diff
@@ -702,6 +702,16 @@ It is pointless to edit the labels, as Renovate bot restores the labels on each
 
 Configure this option if you prefer a different title for the Dependency Dashboard.
 
+## dependencyDashboardVulnerabilitySummary
```

Comment:
If this applies only to OSV vulnerabilities then it seems misnamed/under-documented

Reply:
My expectation has been that we transition osv to the standard way of doing vulnerabilities. If that's not the case, it makes sense to rename it.

Reply:
Dependabot alerts seem preferable when available because they allow dismissal of alerts per repo